test/gc/binding.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+try {
+  module.exports = require('./build/Release/binding').ongc;
+} catch (e) {
+  module.exports = require('./build/Debug/binding').ongc;
```

Comment:
Is it correct to prefer a binding in Release (e.g. if tests are run with `node_g`)? 

For example, build and test Release and then build and test Debug without cleaning in-between.

I don't think it's possible for the `makefile` to build the binding in debug mode (it always runs `node-gyp` with the Release node) -- `vcbuild.bat` runs node-gyp with `"%config%\node"`.